### PR TITLE
Indigo dark theme

### PR DESCRIPTION
**Pull request by sknick**
Dark variant of Indigo. Tested using the widget browser:

image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Review by johnspackman — APPROVED** [review]

**Review by cboulanger — APPROVED** [review]

File: source/class/qx/theme/indigo/AppearanceDark.js [hunk 1]
```diff
@@ -0,0 +1,231 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+   2004-2011 1&1 Internet AG, Germany, http://www.1und1.de
```
Comment (oetiker) [inline source/class/qx/theme/indigo/AppearanceDark.js:8]: guess you should replace this or at least add yours

File: source/class/qx/theme/IndigoDark.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2004-2011 1&1 Internet AG, Germany, http://www.1und1.de
```
Comment (oetiker) [inline source/class/qx/theme/IndigoDark.js:8]: also replace

File: source/class/qx/theme/indigo/ColorDark.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2004-2011 1&1 Internet AG, Germany, http://www.1und1.de
```
Comment (oetiker) [inline source/class/qx/theme/indigo/ColorDark.js:8]: and here

**Review by oetiker — CHANGES_REQUESTED** [review]

**Comment by goldim** [comment]
Is it good idea to extend basic package of qooxdoo with new themes not using (skipping) package system? Btw steps before including to framework descripted [here](https://qooxdoo.org/documentation/v7.2/#/development/contribute?id=coding-the-framework) ?

File: source/class/qx/theme/indigo/AppearanceDark.js (hunk 1 above)
Comment (sknick) [reply source/class/qx/theme/indigo/AppearanceDark.js:8]: I don't have any copyright I would claim, so it's up to the group. I can remove it if you'd prefer?

**Comment by sknick** [comment]
> Is it good idea to extend basic package of qooxdoo with new themes not using (skipping) package system? Btw steps before including to framework descripted [here](https://qooxdoo.org/documentation/v7.2/#/development/contribute?id=coding-the-framework) ?

I think that Qooxdoo desperately needs a dark theme other than TangibleDark as that uses Material UI concepts. As a result, I'd advocate directly putting it into the framework rather than going through the package system as the package system is an extra step that may turn off some users.

Regarding the tests--I assume those are automated by the checks attached to this PR.

File: source/class/qx/theme/indigo/AppearanceDark.js (hunk 1 above)
Comment (sknick) [reply source/class/qx/theme/indigo/AppearanceDark.js:8]: Not trying to be difficult, but I'm saying that I'm not claiming a copyright and don't want to. Does the Qooxdoo organization not have a standard copyright that can be used?

Comment (hkollmann) [reply source/class/qx/theme/indigo/AppearanceDark.js:8]: please change the header to 
```
/* ************************************************************************

   qooxdoo - the new era of web development

   http://qooxdoo.org

   License:
     MIT: https://opensource.org/licenses/MIT
     See the LICENSE file in the project's top-level directory for details.

   Authors:
     * Scott Knick (@sknick)

************************************************************************ */
```

**Review by hkollmann — CHANGES_REQUESTED** [review]

File: source/class/qx/theme/indigo/AppearanceDark.js (hunk 1 above)
Comment (sknick) [reply source/class/qx/theme/indigo/AppearanceDark.js]: Done.

File: source/class/qx/theme/IndigoDark.js (hunk 2 above)
Comment (sknick) [reply source/class/qx/theme/IndigoDark.js]: Copyright removed per Henner's suggestion.

File: source/class/qx/theme/indigo/ColorDark.js (hunk 3 above)
Comment (sknick) [reply source/class/qx/theme/indigo/ColorDark.js]: Copyright removed per Henner's suggestion.

**Review by derrell — APPROVED** [review]

**Review by hkollmann — APPROVED** [review]

**Review by goldim — APPROVED** [review]
Tested on our app. Looks great. No errors and warnings.
Снимок экрана от 2022-12-22 12-55-26

**Comment by goldim** [comment]
I still doubt about adding new themes to qooxdoo framework. It will become bigger and bigger. I know it's not a problem nowadays bc of internet speed and resources of computers. And another moment about naming of qooxdoo dark themes. There are already two of them maybe there are sub namespace needed like dark (folder): qx.theme.dark.Indigo. Btw simple and modern theme in theory also could be made. I doubt only about dark classic theme.

**Comment by hkollmann** [comment]
@oetiker : Is it ok for you now?

**Comment by sknick** [comment]
> I still doubt about adding new themes to qooxdoo framework. It will become bigger and bigger. I know it's not a problem nowadays bc of internet speed and resources of computers. And another moment about naming of qooxdoo dark themes. There are already two of them maybe there are sub namespace needed like dark (folder): qx.theme.dark.Indigo. Btw simple and modern theme in theory also could be made. I doubt only about dark classic theme.

I don't think anyone is proposing adding an indefinite number of themes. In addition, compiled Qooxdoo applications will only include selected themes, not all.

**Review by oetiker — APPROVED** [review]

**Review by level420 — APPROVED** [review]

**Pull request merged**